database/seed.js

```diff
@@ -0,0 +1,25 @@
+const db = require('./index.js');
+
+// create sample data
+const wishesForProductWishlist = [
+  {
+    name:
+    metal:
+    price:
+    imageUrl:
+  }
+];
+
+// need to create seed function that will insert all sample data into database
+const seed = function() {
+  var sql = 'INSERT INTO wishes (name, metal, price, imageUrl) VALUES'
+  db.query(sql, function(err, results) {
+    if (err) {
+      console.error(err);
+    } else {
+      console.log('Seed was successful');
+    }
+  })
+}
+
+seed();
```

Comment:
Tried pulling this down to see if it would work on my machine. Thought that was a good idea, but looks like I can't do that until the PR is accepted? Worth diving into because that would be cool if I could get your code to work on my machine. I would say the structure all looks good, although I'm uncertain about the syntax within the wishesForProductWishList array, no commas or values separating the keys? Is that a MySQL thing? If so, I will (re)learn about it tonight!

P.S. Have we discussed [this](https://www.npmjs.com/package/mysql-db-seed) npm framework? I might give it a whirl for my tables.